### PR TITLE
Minor typos and header rewording

### PR DESCRIPTION
**Pull request by premkumr**

### CODE REVIEW

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md [hunk 1]
```diff
@@ -75,7 +78,7 @@ SELECT * FROM part_9_23 LIMIT 4;
  2023-09-01 00:00:23 | car-2 |    33
 ```
 
-## Retrieval from the parent table
+## Fetching data
```
Comment (aishwarya24) [inline docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:81]:
```suggestion
## Fetch data
```

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md [hunk 2]
```diff
@@ -100,9 +103,9 @@ EXPLAIN ANALYZE SELECT * FROM part_demo;
 
 When querying the parent table, the child partitions are automatically queried.
 
-## Range retrieval
+## Fetching within a time range
```
Comment (aishwarya24) [inline docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:106]:
```suggestion
## Fetch data within a time range
```

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md [hunk 3]
```diff
@@ -136,6 +139,10 @@ DROP TABLE
 Time: 103.214 ms
 ```
 
+## Automating partition management
```
Comment (aishwarya24) [inline docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:142]:
```suggestion
## Automate partition management
```

**Review by aishwarya24 — APPROVED** [review]
LGTM with some suggestions.

File: docs/content/preview/develop/common-patterns/timeseries/global-ordering.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Learn how to distribute your entire time-ordered dataset across tablets and retr
 
 {{<cluster-setup-tabs>}}
 
-## Distributing on timestamp
+## Timestamp based distribution
```
Comment (ddhodge) [inline docs/content/preview/develop/common-patterns/timeseries/global-ordering.md:21]:
```suggestion
## Timestamp-based distribution
```

File: docs/content/preview/develop/common-patterns/timeseries/global-ordering.md [hunk 5]
```diff
@@ -83,7 +83,7 @@ But as the data grows, the tablet splits and one-half moves to a different table
 
 The following section describes how to distribute the ordered data across different tablets.
 
-## Bucketing for distribution
+## Bucket based distribution
```
Comment (ddhodge) [inline docs/content/preview/develop/common-patterns/timeseries/global-ordering.md:86]:
```suggestion
## Bucket-based distribution
```

File: docs/content/preview/develop/common-patterns/timeseries/ordering-by-entity.md [hunk 6]
```diff
@@ -75,7 +75,7 @@ Distributing the data by the entity(`car`) and ordering the data by timestamp fo
 
 The following section describes a basic method to overcome the hot shard problem.
 
-## Bucketing for distribution
+## Bucket based distribution
```
Comment (ddhodge) [inline docs/content/preview/develop/common-patterns/timeseries/ordering-by-entity.md:78]:
```suggestion
## Bucket-based distribution
```

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md [hunk 7]
```diff
@@ -37,7 +37,7 @@ CREATE TABLE part_demo (
 ) PARTITION BY RANGE (ts);
 ```
 
-Create partitions for each month.
+Create partitions for each month. Also, create a `DEFAULT` partition that would hold data that does that fall into any of the other partitions.
```
Comment (ddhodge) [inline docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:40]: "that would hold data that does that fall into any of the other partitions."
that does not fall into?

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md [hunk 8]
```diff
@@ -136,6 +139,10 @@ DROP TABLE
 Time: 103.214 ms
 ```
 
+## Automating partition management
+
+Creation and deletion of partitions can be managed using the [pg_partman](https://github.com/pgpartman/pg_partman) extension or scheduling jobs using [pg_cron](https://github.com/citusdata/pg_cron).
```
Comment (ddhodge) [inline docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:144]: Does YBDB support these extensions?

**Review by ddhodge — COMMENTED** [review]
Extensions supported?

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md (hunk 7 above)
Comment (premkumr) [reply docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:40]: haha. You should have just modified this yourself.

File: docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md (hunk 8 above)
Comment (premkumr) [reply docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:144]: Great catch . will remove this.

Comment (premkumr) [reply docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md:144]: Removed section

**Pull request merged**